gno.land/pkg/sdk/vm/errors.go

```diff
@@ -15,25 +15,31 @@ func (abciError) AssertABCIError() {}
 // declare all script errors.
 // NOTE: these are meant to be used in conjunction with pkgs/errors.
 type (
-	InvalidPkgPathError struct{ abciError }
-	InvalidStmtError    struct{ abciError }
-	InvalidExprError    struct{ abciError }
-	TypeCheckError      struct {
+	InvalidPkgPathError   struct{ abciError }
+	InvalidStmtError      struct{ abciError }
+	InvalidExprError      struct{ abciError }
+	UnauthorizedUserError struct{ abciError }
+	TypeCheckError        struct {
 		abciError
 		Errors []string `json:"errors"`
 	}
 )
 
-func (e InvalidPkgPathError) Error() string { return "invalid package path" }
-func (e InvalidStmtError) Error() string    { return "invalid statement" }
-func (e InvalidExprError) Error() string    { return "invalid expression" }
+func (e InvalidPkgPathError) Error() string   { return "invalid package path" }
+func (e InvalidStmtError) Error() string      { return "invalid statement" }
+func (e InvalidExprError) Error() string      { return "invalid expression" }
+func (e UnauthorizedUserError) Error() string { return "unauthorized user" }
```

Comment:
What do you think about rephrasing this as `unauthorized deployment user`?

Reply:
I maybe wrong, but I don't think it's good idea to add too much context here as it's mean to be a generic error